cypress/integration/pages/articles/testsForAMPOnly.js

```diff
@@ -43,21 +43,80 @@ export const testsThatFollowSmokeTestConfigForAMPOnly = ({
     // `appToggles` tells us whether a feature is toggled on or off in the current environment.
     if (appToggles.mediaPlayer.enabled) {
       describe('Media Player', () => {
-        it('should render a placeholder image', () => {
-          cy.request(`${config[service].pageTypes.articles.path}.json`).then(
-            ({ body }) => {
-              // `video` blocks can also contain audio.
-              const media = getBlockData('video', body);
-              if (media) {
-                cy.get('div[class^="StyledVideoContainer"]').within(() => {
-                  cy.get('amp-img')
-                    .should('have.attr', 'src')
-                    .should('not.be.empty');
-                });
-              }
-            },
-          );
-        });
+        // This test is being temporarily throttled to the service 'news'.
+        if (service === 'news') {
+          it('should have a Media Player', () => {
+            cy.viewport(600, 1008);
+            cy.request(`${config[service].pageTypes.articles.path}.json`).then(
+              ({ body }) => {
+                const media = getBlockData('video', body);
+                if (media) {
+                  cy.get('div[class^="StyledVideoContainer"]').then(() => {
+                    cy.get('iframe[class^="i-amphtml-fill-content"]').then(
```

Comment:
These `get`'s could be merged into one. 🙂

Reply:
Merging this into one breaks the test.

Reply:
Merging this into one breaks the test.